flake.nix

```diff
@@ -6,7 +6,7 @@
     systems.url = "github:nix-systems/default";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/5dcd473408312a874f5666cde92413681fc57418";
```

Comment:
shared-kernel should be updated without changing `flake.nix`; see https://github.com/nammayatri/nammayatri/tree/main/Backend#updating-flake-inputs